AGENTS.md

```
@@ -347,69 +96,24 @@ npm run format && npm run lint
-
-// features/tasks/components/task-form-modal.tsx
-export function TaskFormModal() {
-  return <BaseModal title="Create Task"><TaskForm /></BaseModal>
-}
-```
+1. Search existing components in `src/components/ui/`, `src/components/common/`, `src/features/*/components/`
+2. Extract reusable logic if implementing similar UI patterns multiple times
+3. Avoid duplication - use composition over copy-paste
 
 **Component Organization:**
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)